src/util/settings_persister.js

```diff
@@ -105,6 +105,13 @@ export const persistableFields = [
     type: 'boolean',
     shouldStoreInConfig: true,
   },
+  {
+    category: 'base',
+    name: 'shouldBackup',
+    type: 'boolean',
+    shouldStoreInConfig: true,
+    default: true,
```

Comment:
The PR makes a lot of sense, the implementation works and is exactly modeled after the existing flags.

Unfortunately (like in the literal sense), I cannot merge this PR, yet. The reason is that `default: true` doesn't work. There's no good `default` functionality at this time for settings.

It's possible to set a `default` via `||` in `mapStateToProps` in Settings/index.js, however we can't set the `default` to `|| true`, because that will always evaluate to `true`.

For a new flag/feature, I wouldn't mind merging without a default. If users don't enable it, it's not an issue. However, this flag would by default disable a feature that has been enabled. So, if a user is expeting for the feature to run, it would silently fail her.

Not having a default for Settings has been on my private list for a while. I guess now is the time to think about it. If you've got any ideas on how to implement it, you're welcome to beat me to the punch^^

Reply:
> It's possible to set a `default` via `||` in `mapStateToProps` in Settings/index.js, however we can't set the `default` to `|| true`, because that will always evaluate to `true`.

I'm pretty sure I already addressed that.  Please see https://github.com/200ok-ch/organice/pull/320/files#diff-1260fc7748c44c832b55e6f3369f6fabR219

I'm guessing you are thinking of the existing code which handles defaults for `json` values; however as you can see below, my new code for `boolean` defaults works correctly - or at least it did when I tested it ;-) 

> For a new flag/feature, I wouldn't mind merging without a default. If users don't enable it, it's not an issue. However, this flag would by default disable a feature that has been enabled. So, if a user is expeting for the feature to run, it would silently fail her.

Again, I'm pretty confident that it keeps the existing default behaviour, but ICBW of course.

> Not having a default for Settings has been on my private list for a while. I guess now is the time to think about it. If you've got any ideas on how to implement it, you're welcome to beat me to the punch^^

Well I think that with this PR it now should work for `boolean` and `json` types.  I'm not sure whether it makes sense for `nullable` types, since if the default isn't null, then can the user even nullify it later?